test/__helpers/e2e-system-test.ts

```diff
@@ -158,14 +158,18 @@ export async function e2eTestApp(ctx: ReturnType<typeof setupE2EContext>) {
   // Install a plugin
   const nexusPluginPrismaVersion =
     process.env.NEXUS_PLUGIN_PRISMA_VERSION ?? 'latest'
+  const prismaVersion = process.env.PRISMA_VERSION ?? '2.0.0-preview023'
```

Comment:
This is pretty nasty.

@Weakky once we have more framework plugins I'm _totally_ down to ditch testing prisma here and do others.

The thing is though, this assumes future plugins that use as many features as Prisma but are way simpler to manage, don't double the test time, etc.

Maybe we should create a plugin designed for this, now that users can also learn more, and maybe one taht can be the basis for the plugin scaffolding (`nexus create plugin`).

Actually I think that makes a lot of sense, it does two things at once: test the plugin interface + test `nexus create plugin`.

The e2e test would run that command locally on the system and then go back to the e2e app and install it via file path. Should work!

We just need to update the nexus plugin create scaffolding.

Reply:
Just realizing how much this makes sense. This PR for example should include the update to the scaffolding. It doesn't make sense not to. It has a backwards incompatible change for plugins. Of course the scaffolder should be updated here. Having it just be part of the e2e test would make it impossible for us to forget this in the future.